### PR TITLE
Upgrade dependencies

### PR DESCRIPTION
**Pull request by Igor-J86**

### CODE REVIEW

**Comment by WebDevBring** [comment]
[trello.com/c/trCxlgwm](https://trello.com/c/trCxlgwm)

File: layouts/partials/head.html [hunk 1]
```diff
@@ -29,7 +29,7 @@
     href="https://www.mybring.com/design-system/css/main.min.css?ts={{ now.Unix }}"
   />
   {{ if not $.Site.IsServer }}
-    {{- $jsManifest := getJSON "static/assets/manifest.json" -}}
+    {{- $jsManifest := getJSON "static/assets/.vite/manifest.json" -}}
```
Comment (halvorsanden) [inline layouts/partials/head.html:32]: Any place I can read about this change?

Comment (Igor-J86) [reply layouts/partials/head.html:32]: I cannot find the "why" anywhere, but seems like they just went for it for some reason:
https://github.com/vitejs/vite/pull/14230
https://github.com/vitejs/vite/commit/74fa024

Comment (halvorsanden) [reply layouts/partials/head.html:32]: I see, thanks!

**Pull request merged**